hphp/runtime/server/request-uri.cpp

```diff
@@ -149,7 +149,7 @@ bool RequestURI::process(const VirtualHost *vhost, Transport *transport,
   return true;
 }
 
-void RequestURI::splitURL(String surl, String &base, String &querys) {
+void RequestURI::splitURL(String surl, String &base, String &queries) {
```

Comment:
I don't see any reason not to use the English conjugation. I did try to make sure I didn't corrupt other words, but it's possible. I missed a thing. I am a human and this changeset is indeed larger than what most people can keep in their heads.